ClientTests/SearchTests.swift

```diff
@@ -54,8 +53,8 @@ class SearchTests: XCTestCase {
 
     func testSuggestClient() {
         let webServerBase = startMockSuggestServer()
-
-        let engine = OpenSearchEngine(id: "mock", shortName: "Mock engine", description: nil, image: nil, searchTemplate: "", suggestTemplate: "\(webServerBase)?q={searchTerms}")
+        let engine = OpenSearchEngine(engineId: "mock", shortName: "Mock engine", image: nil, searchTemplate: "", suggestTemplate: "\(webServerBase)?q={searchTerms}",
+                                      isCustomEngine: false)
```

Comment:
Nit: keep args on the same line (or, break them up more so the first line isn't as long)